### PR TITLE
Fix server code style with prettier

### PR DESCRIPTION
**Pull request by jrmi**
First step of ticket #130 

Fix server code style. No other modifications.

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "trailingComma": "all",
+  "tabWidth": 2,
+  "semi": false,
```
Comment (alanfriedman) [inline .prettierrc:4]: Curious, why the preference to omit semicolons? (I'm not necessarily against it)

Comment (jrmi) [reply .prettierrc:4]: Actually, I don't prefer that, but I see this frontend side so I want to be consistent. Tell me what you want and I'll be ok !

Comment (alanfriedman) [reply .prettierrc:4]: Ah, ok! I actually would prefer semicolons on both front and back-end

Comment (jrmi) [reply .prettierrc:4]: Ok for me, I update it as soon as possible.

Comment (jrmi) [reply .prettierrc]: Done

**Pull request merged**